internal/report.go

```diff
@@ -557,3 +558,18 @@ func writeHeading(w *bufio.Writer, s string) {
 		s, "\n",
 		"----------------------------\n"}, ""))
 }
+
+func conversionDuration(conv *Conv, w *bufio.Writer) string {
+	res := ""
+	if conv.DataConversionDuration.Microseconds() != 0 || conv.SchemaConversionDuration.Microseconds() != 0 {
```

Comment:
why is the check for non zero required ? We should be able to print it in any case.